examples/retrieve_priorized_cve_during_the_past_x_days.py

```diff
@@ -0,0 +1,295 @@
+# Script to track priority vulnerabilities in one or more groups dating back X days.  
+# Output is in the form of an HTML page or CSV table.
+from cyberwatch_api import Cyberwatch_Pyhelper
+from datetime import datetime, timedelta, timezone
+import csv
+import urllib3
+import warnings
+
+# Disable SSL warnings
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+# Filter warnings related to unverified HTTPS requests
+warnings.filterwarnings("ignore", message="Unverified HTTPS request")
+
+cyberwatch = Cyberwatch_Pyhelper()
+
+# Here you can select the number of days to be included in the analysis
+# By default it set to 7
+# If days=0, all the CVE are taken into account
+NUMBER_OF_DAY = datetime.now(timezone.utc) - timedelta(days=7)
+
+GROUPS_TO_INCLUDE = [
+    "group1",
+    "group2"
+]
+
+# Function to retrieve the list of unique assets
+def get_all_assets():
+    all_assets = []
+    seen_hostnames = set()
+    page = 1
+    per_page = 500
+
+    while True:  # Infinite loop until no more assets are available

```

Comment:
Can you Initialise assets and use it as your while condition istead of breaking ?